client/config/environment.js

```diff
@@ -4,12 +4,14 @@ var devHost = 'http://localhost:1337';
 var prodHost = 'http://gallery.hairbrain.io';
 var devS3 = 'https://gallery13-dev.s3.amazonaws.com';
 var prodS3 = 'https://gallery13.s3.amazonaws.com';
+var gaId = 'UA-12392326-1';
 
 module.exports = function(environment) {
   var ENV = {
     modulePrefix: 'client',
     podModulePrefix: 'client/pods',
     environment: environment,
+    isProduction: (this.environment === 'production'),
```

Comment:
To be clear, what I'm proposing is:

In config/environment.js:
```js
  const DEBUG_GA = false;
  const GA_ID= 'UA-12392326-1';

  if (environment === 'development') {
    ...
    if (DEBUG_GA) {
      ENV.googleAnalyticsId = GA_ID;
    }
  }
  if (environment === 'test') {
    ...
  }
  if (environment === 'production') {
    ...
    ENV.googleAnalyticsId = GA_ID;
  }
```

And in brocfile:
```
var config = require('./config/environment')(env);

if (config.googleAnalyticsId) {
  app.options.inlineContent = {
    'google-analytics' : {
      file: './ga.js',
        postProcess: function(content) {
        return content.replace(/\{\{GOOGLE_ANALYTICS_ID\}\}/g, config.googleAnalyticsId);
      }
    }
  }
}

```